server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandlerCommonMutations.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.storagehandler;
+
+import com.google.coffeehouse.common.MembershipConstants;
+import com.google.cloud.spanner.Key;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Value;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+* The StorageHandler class holds the functions that add mutations in order to
+* query from the database.
+*/
+
+public class StorageHandlerCommonMutations {
```

Comment:
Same as previous file, please remove this space between the doc comment and the class declaration.